code/iexfeed/iex.q

```diff
@@ -0,0 +1,56 @@
+\d .iex
+
+main_url:@[value;`main_url;"https://api.iextrading.com"];
+convert_epoch:@[value;`convert_epoch;{{"p"$1970.01.01D+1000000j*x}}];
+reqtype:@[value;`reqtype;`both];
+syms:@[value;`syms;`CAT`DOG];
+callback:@[value;`callback;".u.upd"];
+callbackhandle:@[value;`callbackhandle;0i];
+callbackconnection:@[value;`callbackconnection;`];
+quote_suffix:@[value;`quote_suffix;{{[sym] "/1.0/stock/",sym,"/quote"}}];
+trade_suffix:@[value;`trade_suffix;{{[sym]"/1.0/tops/last?symbols=",sym}}];
+upd:@[value;`upd;{{[t;x].iex.callbackhandle(.iex.callback;t; value flip x)}}];
+timerperiod:@[value;`timerperiod;0D00:00:02.000];
+
+init:{[x]
+   if[`main_url in key x;.iex.main_url:x `main_url];
+   if[`quote_suffix in key x;.iex.quote_suffix:x `quote_suffix];
+   if[`trade_suffix in key x;.iex.trade_suffix:x`trade_suffix];
+   if[`syms in key x;.iex.syms: upper x`syms];
+   if[`reqtype in key x;.iex.reqtype:x`reqtype];
+   if[`callbackconnection in key x;.iex.callbackhandle:neg hopen .iex.callbackconnection:x `callbackconnection];
+   if[`callbackhandle in key x;.iex.callbackhandle:x `callbackhandle];
+   if[`callback in key x;.iex.callback: $[.iex.callbackhandle=0; string @[value;x `callback;{[x;y]x set {[t;x]x}}[x`callback]]; x`callback]];
+   if[`upd in key x; .iex.upd:x[`upd]];
+   .iex.timer:$[not .iex.reqtype in key .iex.timer_dict;'`timer;.iex.timer_dict .iex.reqtype];
+   }
+
+get_data:{[main_url;suffix]
+   .Q.hg `$main_url,suffix
+   }
+
+get_last_trade:{tab:{[syms]
+   / This function can run for multiple securities.
+   syms:$[1<count syms;"," sv string[upper syms];string[upper syms]];
+   / Construct the GET request
+   suffix:.iex.trade_suffix[syms];
+   / Parse json response and put into table. Trade data from https://iextrading.com/developer/
+   data:.j.k .iex.get_data[.iex.main_url;suffix];
+   tab:select sym:`$symbol, price:`float$price, size:`int$size, stop:(count data)#0b, cond:(count data)#`char$(), ex:(count data)#`char$(), srctime:"P"$string(.iex.convert_epoch time) from data
```

Comment:
`srctime:"P"$string(.iex.convert_epoch time)`

is the string and cast necessary here? looks like `.iex.convert_epoch` returns a timestamp?

```
q).iex.convert_epoch 1517605625352
2018.02.02D21:07:05.352000000
```

(same in the `get_quote` function below)